src/robotide/context/__init__.py

```diff
@@ -248,11 +260,11 @@ def bind_keys_to_evt_menu(target, actions):
         <td>Paste (does not move cells or rows)</td>
     </tr>
     <tr>
-        <td>Shift-CtrlCmd-V</td>
+        <td>CtrlCmd-Shift-V</td>
         <td>Insert (adds empty rows and pastes data)</td>
     </tr>
     <tr>
-        <td>Delete</td>
+        <td>Del</td>
```

Comment:
Del is an abbreviation, why not using Delete?

Reply:
On the grid cell menu the description of the shortcut is also Del. I followed the same syntax, same order to get the same collection.